src/deterministicArchive.js

```diff
@@ -22,20 +23,21 @@ async function resolveFilesRecursiveForDir(dirOrFile) {
   const isDir = (await fs.promises.lstat(resolvedDirOrFile)).isDirectory();
 
   if (isDir) {
-    const files = await fs.promises.readdir(resolvedDirOrFile, {
-      withFileTypes: true,
-      recursive: true,
+    const files = Array.from(
+      await glob.sync('**/*', {
```

Comment:
`await glob.sync` is a bit strange for a couple of reasons:

1. `globSync` is a named export of the glob package, that I think we'd want instead of this if we want synchronous mode
2. `await` a sync function is unnecessary
3. Since we are in an async function can we use the async glob?

Reply:
Yeah I kept `glob.sync` because of issues locally. Turns out I hadn't `yarn install`ed in a while.